app/src/main/java/app/revanced/manager/ui/viewmodel/MainViewModel.kt

```diff
@@ -52,7 +53,7 @@ class MainViewModel(
     }
 
     private suspend fun checkForManagerUpdates() {
-        if (!prefs.managerAutoUpdates.get() || !networkInfo.isConnected()) return
+        if (BuildConfig.DEBUG || BuildConfig.VERSION_NAME.contains("-dev") || !prefs.managerAutoUpdates.get() || !networkInfo.isConnected()) return
```

Comment:
The code look too long for me

Prehaps something like this could work

```suggestion
        if (BuildConfig.DEBUG || 
            BuildConfig.VERSION_NAME.contains("-dev") || 
            !prefs.managerAutoUpdates.get() || 
            !networkInfo.isConnected()) return
```